### PR TITLE
1.0.4 patch train updates

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Will need to coordinate with @dougbu for https://github.com/aspnet/Mvc/pull/5782

### CODE REVIEW

**Comment by dougbu** [comment]
@JunTaoLuo I'm just waiting for @pranavkm on #5782.

But, why are we calling this 1.0.4? Thought MVC was on the high end with the latest patch branch being `rel/1.0.3`.

**Comment by JunTaoLuo** [comment]
It's the 1.0.4 patch train but the MVC packages will be 1.0.3.

File: src/Microsoft.AspNetCore.Mvc.Core/project.json [hunk 1]
```diff
@@ -20,22 +20,22 @@
...
-    "Microsoft.AspNetCore.Routing": "1.0.2",
+    "Microsoft.AspNetCore.Authorization": "1.0.2",
+    "Microsoft.AspNetCore.Hosting.Abstractions": "1.0.2",
+    "Microsoft.AspNetCore.Mvc.Abstractions": "1.0.3",
+    "Microsoft.AspNetCore.Http": "1.0.2",
+    "Microsoft.AspNetCore.Routing": "1.0.3",
     "Microsoft.AspNetCore.Routing.DecisionTree.Sources": {
       "type": "build",
-      "version": "1.0.2"
+      "version": "1.0.3"
```
Comment (dougbu) [inline src/Microsoft.AspNetCore.Mvc.Core/project.json]: Is this the only *.Sources package that's been updated?

**Comment by dougbu** [comment]
> Will need to coordinate with @dougbu for #5782

All done. @JunTaoLuo that PR is now in the `rel/1.0.3` branch.

File: src/Microsoft.AspNetCore.Mvc.Core/project.json (hunk 1 above)
Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore.Mvc.Core/project.json]: There are a few, you can see all the package version updates in https://github.com/aspnet/Coherence-Patch/pull/40/files#diff-b68e642d223a5f81dc260548f1e0df64. 

The *.Sources packages I have seen update are
- Microsoft.AspNetCore.DataProtection.Sources
- Microsoft.AspNetCore.DiagnosticsViewPage.Sources
- Microsoft.AspNetCore.Razor.Test.Sources
- Microsoft.AspNetCore.Routing.DecisionTree.Sources

Any particular concerns regarding sources packages?

Comment (dougbu) [inline src/Microsoft.AspNetCore.Mvc.Core/project.json]: > Any particular concerns regarding sources packages?

No. I just noticed this was the only .Sources dependency you updated. Your answer makes sense.

**Comment by dougbu** [comment]
@JunTaoLuo you need to rebase this branch. No need to re-add the SDK version to global.json for example.

File: global.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "projects": ["src", "test/WebSites", "samples"],
     "sdk": {
-      "version": "1.0.0-preview2-003154"
+        "version": "1.0.0-preview2-003154"
```
Comment (pranavkm) [inline global.json]: Undo.

Comment (JunTaoLuo) [inline global.json]: Sigh... inconsistent tabs even within the same file.

**Pull request merged**